tests/yam/migration/migration_unattended.pm

```diff
@@ -22,20 +23,27 @@ sub run {
 
     my $repo_home = "http://download.suse.de/ibs/home:/fcrozat:/SLES16/SLE_\$releasever";
     my $repo_images = 'http://download.suse.de/ibs/home:/fcrozat:/SLES16/images/';
-    zypper_call("ar --refresh -p 90 '$repo_home' home_sles16");
-    zypper_call("ar --refresh -p 90 $repo_images home_images");
+    zypper_call("ar --refresh -p 95 '$repo_home' home_sles16");
```

Comment:
keep 90, for development we should follow this (as far as it is up to date):
https://confluence.suse.com/spaces/SLES16/pages/1654456421/SLES+16+Migration